lib/rouge/lexers/factor.rb

```diff
@@ -243,7 +243,7 @@ def self.builtins
         end
 
         # strings
-        rule %r/"""\s+.*?\s+"""/, Str
+        rule %r/"""[^"].*?[^"]"""/, Str
```

Comment:
Actually, I think this is wrong, let me fix that.

Reply:
Maybe it's not even supported: https://github.com/factor/factor/blob/master/core/strings/strings-tests.factor

Reply:
Interesting. Looks like GitHub is parsing the code correctly'ish 🤔 The highlight for double quotes seems to be out of place but overall it looks good.

I was suggesting lifting the regexes from `Pygments`. However, it seems Pygments 2.8.1 (latest), the Python package that Rouge compatible with does not handle triple quotes well (could be an existing bug).

<img width="736" alt="Screen Shot 2021-03-11 at 8 17 00 pm" src="https://user-images.githubusercontent.com/756722/110764468-36794a80-82a7-11eb-852e-f693529fa562.png">

Reply:
According to https://github.com/factor/factor/issues/1474, it seems like triple quotes are not supported.